card/lib/card/auth.rb

```diff
@@ -214,7 +213,9 @@ def always_ok_usr_id? usr_id
       def createable_types
         type_names = Auth.as_bot do
           Card.search type: Card::CardtypeID, return: :name,
-                      not: { codename: ['in'] + NON_CREATEABLE_TYPES }
+                      not: {
+                        codename: ['in'] + Card.config.non_createable_types
+                      }
```

Comment:
How can I change a class variable in a mod/deck? I tried to change NON_CREATEABLE_TYPES in wikirate but it didn't work